api/inventorypb/agents.proto

```diff
@@ -325,6 +325,28 @@ message RDSExporter {
   bool enhanced_metrics_disabled = 10;
 }
 
+// ExternalExporter runs on any Node type, including Remote Node.
+message ExternalExporter {
+  // Unique randomly generated instance identifier.
+  string agent_id = 1;
+  // Node identifier where this instance runs.
+  string runs_on_node_id = 2;
+  // If disabled, metrics from this exporter will not be collected.
+  bool disabled = 3;
+  // Service identifier.
+  string service_id = 4;
+  // HTTP basic auth username for collecting metrics.
+  string username = 5;
+  // Scheme to generate URI to exporter metrics endpoints(default: http).
```

Comment:
But that structure is returned, and that field is always set, isn't it? So there is no default?